### PR TITLE
Update the design of the focal point handle

### PR DESCRIPTION
**Pull request by mtias**
Our older focus point ring is both a bit date and suffering from over precision in its presentation.

Before:
image

After:
image

### CODE REVIEW

File: packages/components/src/focal-point-picker/styles/focal-point-style.ts [hunk 1]
```diff
@@ -3,45 +3,30 @@
...
 	cursor: grab;
-	height: 30px;
-	margin: -15px 0 0 -15px;
-	opacity: 0.8;
+	height: 48px;
+	margin: -24px 0 0 -24px;
 	position: absolute;
 	user-select: none;
-	width: 30px;
+	width: 48px;
```
Comment (talldan) [inline packages/components/src/focal-point-picker/styles/focal-point-style.ts:13]: What do you think about using something like `clip-path: circle( 50% );` here as well? At the moment you can mouseover the invisible parts of the rectangle and still drag. It's quite nice when only the circle is draggable.

**Review by talldan — COMMENTED** [review]
Looks good. I'm not sure how to check it's sufficient contrast, but it is noticeably bigger than then previous one.

I noticed it can overlap the labels below it, which seems like something that might need some tweaking:
Screen Shot 2022-10-18 at 3 04 32 pm

File: packages/components/src/focal-point-picker/styles/focal-point-style.ts (hunk 1 above)
Comment (mtias) [reply packages/components/src/focal-point-picker/styles/focal-point-style.ts:13]: We can try it!

**Comment by mtias** [comment]
It's fine that it can overlap labels, it's just a matter of size.

File: packages/components/src/focal-point-picker/styles/focal-point-style.ts (hunk 1 above)
Comment (mtias) [reply packages/components/src/focal-point-picker/styles/focal-point-style.ts:13]: It causes issues with the shadow which is necessary for contrast in more varied backgrounds

Comment (stokesman) [reply packages/components/src/focal-point-picker/styles/focal-point-style.ts:13]: One way to make Dan’s suggestion work with the shadow is by putting the shadow and border-radius on the parent with `pointer-events: none` and the inner element gets the clip-path and `pointer-events: auto`.

File: packages/components/src/focal-point-picker/styles/focal-point-style.ts [hunk 2]
```diff
@@ -3,45 +3,30 @@
...
 `;
 
-export const PointerIconSVG = styled( SVG )`
+export const PointerCircle = styled.div`
 	display: block;
 	height: 100%;
 	left: 0;
 	position: absolute;
 	top: 0;
 	width: 100%;
```
Comment (stokesman) [inline packages/components/src/focal-point-picker/styles/focal-point-style.ts]:
```suggestion
	height: 100%;
```
Just a little simplification that could be done while we are looking at these styles. I'd even suggest getting rid of the this inner element altogether if there's no interest in making the `clip-path` suggestion of Dan’s work.

File: packages/components/src/focal-point-picker/styles/focal-point-style.ts [hunk 3]
```diff
@@ -3,45 +3,21 @@
...
-
-export const FocalPointWrapper = styled.div`
+export const PointerCircle = styled.div`
 	background-color: transparent;
 	cursor: grab;
-	height: 30px;
-	margin: -15px 0 0 -15px;
-	opacity: 0.8;
+	height: 48px;
+	margin: -24px 0 0 -24px;
```
Comment (jasmussen) [inline packages/components/src/focal-point-picker/styles/focal-point-style.ts:10]: Nice base12 values!

**Review by jasmussen — APPROVED** [review]
I've been wanting to update this ring for ages, and this feels like a great step forward with a big and highly grabbable footprint. It's working well for me:

focus

The removal of the SVG and the code simplification is also much appreciated. The combined blur, color, shadow, and size, IMO makes it very clearly distinguishable from the background, I think we should try this, and I'd be happy to catch any followups on this one, in case there are any.

**Pull request merged**